src/oc/storage/api/orgs.clj

```diff
@@ -230,7 +238,8 @@
   ;; Authorization
   :allowed? (by-method {
     :options true
-    :get (fn [ctx] (access/access-level-for conn slug (:user ctx)))
+      :get (fn [ctx] (let [user (user-from-context ctx)]
```

Comment:
@thepug indentation seems off here

Reply:
@bago2k4 just a reminder, you should just fix these as a reviewer